.circleci/config.yml

```diff
@@ -88,7 +88,15 @@ jobs:
     steps:
       - checkout
       - run_tests:
-          test_type: unit
+          test_type: test-unit
+
+  lint:
```

Comment:
@Btodhunter this didn't seem like it got picked up in the current PR. Is this something that is going to be triggered *after* this is merged? I could swear I've seen configs to be executed in the same PR that implements them

Reply:
It will run on the same PR that implements it, you just need to add a lint job to the workflows.